basenode/src/main/java/io/coti/basenode/controllers/BaseNodeAdminController.java

```diff
@@ -73,15 +70,39 @@ public ResponseEntity<IResponse> backup() {
         return dbRecoveryService.manualBackupDB();
     }
 
-    @Operation(summary = "Set Node Fee", operationId = "setFeeValue")
+    @Operation(summary = "Set Node Constant Fee", operationId = "setFeeValue")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "OK",
                     content = {@Content(mediaType = "application/json",
                             schema = @Schema(implementation = GetNodeFeesDataResponse.class))}),
             @ApiResponse(responseCode = "400", description = "Invalid request parameters.",
                     content = @Content)})
-    @PostMapping(path = "/fee/set")
-    public ResponseEntity<IResponse> setFeeValue(@Valid @RequestBody NodeFeeSetRequest nodeFeeSetRequest) {
-        return nodeFeesService.setFeeValue(nodeFeeSetRequest);
+    @PostMapping(path = "/fee/set/constant")
+    public ResponseEntity<IResponse> setFeeValue(@Valid @RequestBody ConstantTokenFeeSetRequest constantTokenFeeSetRequest) {
+        return nodeFeesService.setFeeValue(constantTokenFeeSetRequest);
+    }
+
+    @Operation(summary = "Set Node Ratio Fee", operationId = "setFeeValue")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK",
+                    content = {@Content(mediaType = "application/json",
+                            schema = @Schema(implementation = GetNodeFeesDataResponse.class))}),
+            @ApiResponse(responseCode = "400", description = "Invalid request parameters.",
+                    content = @Content)})
+    @PostMapping(path = "/fee/set/ratio")
+    public ResponseEntity<IResponse> setFeeValue(@Valid @RequestBody RatioTokenFeeSetRequest ratioTokenFeeSetRequest) {
+        return nodeFeesService.setFeeValue(ratioTokenFeeSetRequest);
+    }
+
+    @Operation(summary = "Delete Node Ratio Fee", operationId = "deleteFeeValue")
```

Comment:
Is this summary fully reflects the functionality?

Reply:
swagger annotations can mess up the code. maybe consider taking it out from the controller itself.

Reply:
resolved